contracts/governance/Vesting/VestingRegistryLogic.sol

```diff
@@ -289,6 +308,41 @@ contract VestingRegistryLogic is VestingRegistryStorage {
         return vestings[uid].vestingAddress;
     }
 
+    /**
+     * @dev check if the specific vesting address is team vesting or not
+     * @dev read the vestingType from vestingDetail storage
+     *
+     * @param _vestingAddress address of vesting contract
+     *
+     * @return true for teamVesting, false for normal vesting
+     */
+    function isTeamVesting(address _vestingAddress) external view returns (bool) {
+        if (
+            vestingDetail[_vestingAddress].vestingAddress == address(0) ||
+            vestingDetail[_vestingAddress].vestingType != uint256(VestingType.TeamVesting)
```

Comment:
```suggestion  
            vestingDetails[_vestingAddress].vestingAddress == address(0) ||
            vestingDetails[_vestingAddress].vestingType != uint256(VestingType.TeamVesting)
```

Reply:
updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/447/commits/79f72c867c17cd1f12ed1e726ee79e2c308a8e9e)